Install libqgpgme as libqgpgme4 to not conflict with security/gpgme-qt5

--- qgpgme/CMakeLists.txt.orig	2016-11-19 14:44:57 UTC
+++ qgpgme/CMakeLists.txt
@@ -49,6 +49,7 @@ if ( _any_gpgme_found )
    set_target_properties( qgpgme PROPERTIES
      VERSION ${_qgpgme_version}
      SOVERSION ${_qgpgme_soversion}
+     OUTPUT_NAME qgpgme4
    )
 
    install( TARGETS qgpgme EXPORT kdepimlibsLibraryTargets ${INSTALL_TARGETS_DEFAULT_ARGS})
